doc/userguide/unix-socket.rst

```diff
@@ -63,6 +63,11 @@ The set of existing commands is the following:
 * capture-mode: display capture system used
 * conf-get: get configuration item (see example below)
 * dump-counters: dump Suricata's performance counters
+* ruleset-reload-rules: reload ruleset and wait for completion
+* ruleset-reload-nonblocking: ask a ruleset reload and return
```

Comment:
I would write **trigger** or **request** a ruleset reload. Or copy the part above and write **reload rulesat and proceed without waiting**.